jax/_src/numpy/ufuncs.py

```diff
@@ -241,13 +241,47 @@ def bitwise_right_shift(x1: ArrayLike, x2: ArrayLike, /) -> Array:
     np.issubdtype(x1.dtype, np.unsignedinteger) else lax.shift_right_arithmetic
   return lax_fn(x1, x2)
 
-@implements(np.absolute, module='numpy')
+
 @partial(jit, inline=True)
 def absolute(x: ArrayLike, /) -> Array:
+  r"""Calculate the absolute value element-wise.
+
+  LAX-backend implementation of :code:`numpy.absolute()`.
+
+  This is the same function as :code:`jnp.abs`.
+
+  Args:
+    x: Input array
+
+  Returns:
+    An array-like object containing the absolute value of each element in :math:`x`,
+    with the same shape as :math:`x`. For complex valued input, :math:`a + ib`,
+    the absolute value is :math:`\sqrt{a^2+b^2}`.
+
+  Example:
+    >>> x1 = jnp.array([5, -2, 0, 12])
+    >>> jnp.absolute(x1)
+    Array([ 5,  2,  0, 12], dtype=int32)
+
+    >>> x2 = jnp.array([[ 8, -3, 1],[ 0, 9, -6]])
+    >>> jnp.absolute(x2)
+    Array([[8, 3, 1],
+           [0, 9, 6]], dtype=int32)
+
+    >>> x3 = jnp.array([3 + 2j, 3 - 4j, -5 + 0j])
+    >>> jnp.absolute(x3)
+    Array([3.6055512, 5.       , 5.       ], dtype=float32)
+  """
   check_arraylike('absolute', x)
   dt = dtypes.dtype(x)
   return lax.asarray(x) if dt == np.bool_ or dtypes.issubdtype(dt, np.unsignedinteger) else lax.abs(x)
-abs = implements(np.abs, module='numpy')(absolute)
+
+
+@partial(jit, inline=True)
+def abs(x: ArrayLike, /) -> Array:
+  """Alias of :func:`jax.numpy.absolute`
+  """
```

Comment:
Add a period, skip the newline:
```suggestion
  """Alias of :func:`jax.numpy.absolute`."""
```

Reply:
Hmm.. just saw this, CI seems to expect the "Args:" part, would this formatting skip that check?

Reply:
Oh yeah, we can add this to the skips in that test. No need to list args in an alias